### DISCUSSION TITLE
Number Shader for easier setup

### DISCUSSION THREAD

**Discussion by AndyK84** [200Tigersbloxed/UnityMods#17] [Ideas]
A number shader like this: https://www.patreon.com/posts/simple-counter-62864361
may be worth looking into to simplify the setup and tone down the material usage a bit of you want to have more then one HR setup on the same Avi.

**Comment by 200Tigersbloxed** [comment]
Hi!

Prefab v2, which is currently in-testing (on our [discord](https://fortnite.lol/discord)), has a simple setup process, regardless of how many counters there are, achieves this.

I have chosen to stray away from the use of shaders to display numbers, because they aren't supported on Quest. While the shader may be able to render on Android, VRChat only prohibits use of their built-in mobile shaders. If you'd like to adapt your counter to use shaders instead, that's totally fine, however I will not be implementing the shaders into the prefab for now.

Prefab v2 allows for a simple setup and universal support for a counter on an avatar, which is why I prefer it over shading, which is not universal and *probably* not as customizable as Prefab v2.

**Reply by AndyK84** [reply]
Fair point. Nothing to add to it :-D
